R/qra_fit_convenience.R

```diff
@@ -98,6 +100,7 @@ load_covid_forecasts_relative_horizon <- function(
   hub_repo_path,
   data_processed_subpath = "data-processed/",
   monday_dates,
+  as_of,
```

Comment:
provide default value of NULL
```suggestion
  as_of = NULL,
```